sdk/tests/conformance2/textures/misc/copy-texture-image-webgl-specific.html

```diff
@@ -174,6 +174,140 @@
     gl.deleteTexture(texture[1]);
 };
 
+function copytexsubimage3d_test(xx, yy, copyWidth, copyHeight) {
```

Comment:
Could you rename this to something clearer like copytexsubimage3d_out_of_bounds_test_helper?